src/lib/yang/snabb-softwire-v2.yang

```diff
@@ -26,13 +21,117 @@ module snabb-softwire-v2 {
        This also removes the psid-map list and adds a new port-set
        container on the softwire container instead. This will help
        adding the softwires as well as bring it more inline with the
-       ietf-softwire schema.";
+       ietf-softwire schema.
+
+       The addition of /softwire-config/instance allows for configuring
+       multiple instances of the lwAFTR with a shared binding table and
+       other common configuration properties.";
+  }
+
+  revision 2016-11-04 {
+    description
+     "Initial revision.";
   }
 
   container softwire-config {
     description
      "Configuration for Snabb lwaftr.";
 
+    list instance {
+      description
+       "Provides configuration for specific instances of the lwAFTR.
+        These configuration options will only affect the specific lwaftr
+        with the given name specified in the name leaf. The other options
+        not present in this list are shared amongst all instances.";
+
+      key "name";
+
+      leaf name {
+        type string;
+        description
+         "Name of lwAFTR instance. This must be unique amongst the Snabb
```

Comment:
Here I think we want one name for the whole tree of lwaftr processes, because it will be a single endpoint from a "snabb config" perspective.